frontend/src/environments/environment.ts

```diff
@@ -4,9 +4,7 @@
 
 export const environment = {
   production: false,
-  // apiUrl: 'http://localhost:3001',
-  apiUrl: 'http://159.203.86.45:3001',
-  // apiUrl: "https://mips-api.makerdao.com",
+  apiUrl: "https://mips-api-staging.makerdao.com",
```

Comment:
@eskp if now both staging and prod are running in K8s, the frontends can connect internally to the backends via `http://backend:3000`, so I think we can put this value for `apiUrl` in all 3 files under `/environment/`

Reply:
In case you want to apply the change suggested by Cristi:
```suggestion
  apiUrl: 'http://backend:3000',
```

Reply:
Thanks guys but the API needs to be accessible externally by the browser frontend.